### ISSUE TITLE
weaving in practical guide from moz study groups

### ISSUE THREAD

**Issue by auremoser** [OKScienceDE/Open_Science_101#51]
I'm really excited about this project and would love to help development here.

Perhaps it's already been suggested, but there's also a little outline for an [Open Science Utility Belt or Primer](https://github.com/mozillascience/studyGroupLessons/issues/7), developed by @BillMills.

Would love to help or add a label to this issue but sadly I that isn't an option, maybe because I'm not a core contributors; anyway, great work!

**Comment by aleimba** [comment]
Thanks @auremoser haven't seen that yet! Don't have a label for it, but it definitely belongs to our "Check out what's already there" issue #5 !

**Comment by auremoser** [comment]
Cool, let us know how we can help. We've developed a template for curriculum on gh-pages that we used for our working open curriculum: 
[repo](https://github.com/mozillascience/working-open-workshop)
[homepage/site](http://mozillascience.github.io/working-open-workshop/)
[lesson handout example](http://mozillascience.github.io/working-open-workshop/data_reuse/)

I also forked it for [Space Apps 2016](http://sciencehackdayny.github.io/data-camp-16/), and of of our community members did the same for his [github account](http://blog.jasonbobe.net/sharing-guidebook/). Super easy, happy to help adapt that when you're ready with your outline. :) No pressure though.

**Comment by aleimba** [comment]
yes, we used your great WOW resource extensively for our PERSONAS, Code of Conduct etc. (@kaythaney suggested it also already: #30).

But this gh-pages fork is very cool. Actually, creating a website is one of our future goals (#16) ;-).

**Comment by auremoser** [comment]
Hurray!

**Comment by aleimba** [comment]
@auremoser, I think you're WOW course is under CC-BY, however we want to have all our stuff under CC0 (https://github.com/OKScienceDE/Open_Science_101#license) to make it as open as possible. So we can't just fork your gh-pages, but we'll definitely put it to our ideas collection! Thanks again!

**Comment by auremoser** [comment]
Oh I'm sorry, @aleimba, but thanks for considering it.
